source/_integrations/reolink.markdown

```diff
@@ -171,6 +171,7 @@ The following models have been tested and confirmed to work:
 - C2 Pro
 - E1 Zoom
 - E1 Outdoor
+- E1 Outdoor Pro (Wi-Fi)
```

Comment:
There is only one variant of the E1 Outdoor Pro, so no need to specify Wi-Fi, that is only used if there are 2 seperate variants.

```suggestion
- E1 Outdoor Pro
```

Reply:
You are right, thanks for updating.